ValveResourceFormat/IO/FileExtract.cs

```diff
@@ -182,6 +129,17 @@ public static ContentFile Extract(Resource resource)
             return contentFile;
         }
 
+        public static bool IsChildResource(Resource resource)
+        {
+            if (resource.EditInfo is ResourceEditInfo2 redi2)
+            {
+                return redi2.SearchableUserData.GetProperty<long>("IsChildResource") == 1;
+            }
+
+            var extraIntData = (Blocks.ResourceEditInfoStructs.ExtraIntData)resource.EditInfo.Structs[ResourceEditInfo.REDIStruct.ExtraIntData];
+            return extraIntData.List.Where(x => x.Name == "IsChildResource").Select(x => x.Value).FirstOrDefault() == 1;
```

Comment:
```suggestion
            return extraIntData.List.FirstOrDefault(x => x.Name == "IsChildResource")?.Value == 1;
```

I didn't test it, but you should be able to get something like this to work.